client/providers/OmniChannelProvider.tsx

```diff
@@ -96,6 +96,7 @@ const OmnichannelEnabledProvider: FC = ({ children }) => {
 		setContextValue((context) => ({
 			...context,
 			agentAvailable: user?.statusLivechat === 'available',
+			...routeConfig && { routeConfig },
```

Comment:
You don't need this, you can rely on `omnichannel contexts` -> https://github.com/RocketChat/Rocket.Chat/blob/develop/client/contexts/OmnichannelContext.ts#L22

Reply:
https://github.com/RocketChat/Rocket.Chat/pull/21285#discussion_r602498056